### ISSUE TITLE
Can't build version 0.7.0 with conan

### ISSUE THREAD

**Issue by rbrugo** [BobSteagall/wg21#59]
Installing `linear_algebra` with conan fails with the following error:
```
$ conan install -r linear_algebra linear_algebra/0.7.0@public-conan/testing --build=missing
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=20
compiler.libcxx=libstdc++11
compiler.version=10.2
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

linear_algebra/0.7.0@public-conan/testing: WARN: Package is corrupted, removing folder: /home/rbrugo/.conan/data/linear_algebra/0.7.0/public-conan/testing/package/ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb
Installing package: linear_algebra/0.7.0@public-conan/testing
Requirements
    linear_algebra/0.7.0@public-conan/testing from 'linear_algebra' - Cache
Packages
    linear_algebra/0.7.0@public-conan/testing:ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb - Build

Installing (downloading, building) binaries...
linear_algebra/0.7.0@public-conan/testing: WARN: Build folder is dirty, removing it: /home/rbrugo/.conan/data/linear_algebra/0.7.0/public-conan/testing/build/ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb
linear_algebra/0.7.0@public-conan/testing: Copying sources to build folder
linear_algebra/0.7.0@public-conan/testing: Building your package in /home/rbrugo/.conan/data/linear_algebra/0.7.0/public-conan/testing/build/ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb
linear_algebra/0.7.0@public-conan/testing: Generator cmake created conanbuildinfo.cmake
linear_algebra/0.7.0@public-conan/testing: Calling build()
-- The C compiler identification is GNU 10.2.0
-- The CXX compiler identification is GNU 10.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan setting CPP STANDARD: 20 WITH EXTENSIONS OFF
-- Conan: Compiler GCC>=5, checking major version 10.2
-- Conan: Checking correct version: 10.2
-- Conan: C++ stdlib: libstdc++11
Scanning dependencies of target mdspan-populate
[ 11%] Creating directories for 'mdspan-populate'
[ 22%] Performing download step (git clone) for 'mdspan-populate'
Cloning into 'mdspan-src'...
fatal: invalid reference: master
CMake Error at mdspan-subbuild/mdspan-populate-prefix/tmp/mdspan-populate-gitclone.cmake:40 (message):
  Failed to checkout tag: 'master'


make[2]: *** [CMakeFiles/mdspan-populate.dir/build.make:111: mdspan-populate-prefix/src/mdspan-populate-stamp/mdspan-populate-download] Error 1
make[1]: *** [CMakeFiles/Makefile2:95: CMakeFiles/mdspan-populate.dir/all] Error 2
make: *** [Makefile:103: all] Error 2

CMake Error at /usr/share/cmake-3.18/Modules/FetchContent.cmake:987 (message):
  Build step for mdspan failed: 2
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FetchContent.cmake:1082:EVAL:2 (__FetchContent_directPopulate)
  /usr/share/cmake-3.18/Modules/FetchContent.cmake:1082 (cmake_language)
  /usr/share/cmake-3.18/Modules/FetchContent.cmake:1125 (FetchContent_Populate)
  CMakeLists.txt:35 (FetchContent_MakeAvailable)


-- Configuring incomplete, errors occurred!
See also "/home/rbrugo/.conan/data/linear_algebra/0.7.0/public-conan/testing/build/ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb/CMakeFiles/CMakeOutput.log".
linear_algebra/0.7.0@public-conan/testing: 
linear_algebra/0.7.0@public-conan/testing: ERROR: Package 'ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb' build failed
linear_algebra/0.7.0@public-conan/testing: WARN: Build folder /home/rbrugo/.conan/data/linear_algebra/0.7.0/public-conan/testing/build/ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb
ERROR: linear_algebra/0.7.0@public-conan/testing: Error in build() method, line 48
	self.cmake.build()
while calling 'cmake', line 38
	self._cmake.configure()
	ConanException: Error 1 while executing cd '/home/rbrugo/.conan/data/linear_algebra/0.7.0/public-conan/testing/build/ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb' && cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE="Release" -DCONAN_CMAKE_CXX_STANDARD="20" -DCONAN_CMAKE_CXX_EXTENSIONS="OFF" -DCONAN_STD_CXX_FLAG="-std=c++2a" -DCONAN_IN_LOCAL_CACHE="ON" -DCONAN_COMPILER="gcc" -DCONAN_COMPILER_VERSION="10.2" -DCONAN_CXX_FLAGS="-m64" -DCONAN_SHARED_LINKER_FLAGS="-m64" -DCONAN_C_FLAGS="-m64" -DCONAN_LIBCXX="libstdc++11" -DCMAKE_INSTALL_PREFIX="/home/rbrugo/.conan/data/linear_algebra/0.7.0/public-conan/testing/package/ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb" -DCMAKE_INSTALL_BINDIR="bin" -DCMAKE_INSTALL_SBINDIR="bin" -DCMAKE_INSTALL_LIBEXECDIR="bin" -DCMAKE_INSTALL_LIBDIR="lib" -DCMAKE_INSTALL_INCLUDEDIR="include" -DCMAKE_INSTALL_OLDINCLUDEDIR="include" -DCMAKE_INSTALL_DATAROOTDIR="share" -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY="ON" -DCONAN_EXPORTED="1" -DLA_BUILD_PACKAGE="True" -DLA_ENABLE_TESTS="False" -Wno-dev '/home/rbrugo/.conan/data/linear_algebra/0.7.0/public-conan/testing/build/ce08cfd0c0f8c5c13a09c972c6737a498d5c23fb'
```

**Comment by BobSteagall** [comment]
Try updating the branch you've checked out (presumably master) with a git pull.   The mdspan project changed the name of its primary branch from "master" to "stable" a few months ago.   

We adjusted this project for this change with this commit: https://github.com/BobSteagall/wg21/commit/e19ba8f0823b361517659b59f8781806ea79f320

**Comment by rbrugo** [comment]
I can see that version 0.7.1 has this change, but it seems that version 0.7.0 does not. Editing the CMakeLists.txt exported from conan solves the problem

**Comment by BobSteagall** [comment]
Are you working in the "r7" branch?

**Comment by rbrugo** [comment]
I'm not sure, I'm just launching
```
conan install -r linear_algebra linear_algebra/0.7.0@public-conan/testing --build=missing
```
from my terminal

**Comment by Twon** [comment]
Hi, thanks for reporting this issue. I've released new `linear_algebra/0.7.0@public-conan/testing` and `linear_algebra/0.7.0@public-conan/stable` packages to address this.  I'll close this issue for now.  Please reopen or file a new issue if you run into any further problems.
